runtime-decompiler/src/main/java/org/jrd/frontend/PluginMangerFrame/FileSelectorArrayRow.java

```diff
@@ -39,7 +39,7 @@
         });
 
         chooser = new JFileChooser();
-        File dir = new File(Directories.getPluginDirectory());
+        File dir = new File(System.getProperty("user.home") + File.separator + ".m2" + File.separator + "repository");
```

Comment:
.m2? What other options o we have? Home? downloads?
On contrary, .m2 can easily  be went  up....

Reply:
Maven's local repository is where the jars are expected to be if the user downloads them via the downloadPlugins plugin.

Reply:
yy, I failed to find better lcoation to. Maybe add fallback as in abvoe duplicated code? (to cal the same new method, you are going to create :) ... with test :) )

Reply:
Yes, fallback is a good idea ':D